doc/source/cluster/kubernetes/getting-started/rayservice-quick-start.ipynb

```diff
@@ -369,6 +369,22 @@
    "source": [
     "When the Ray Serve applications are healthy and ready, KubeRay creates a head service and a Ray Serve service for the RayService custom resource. For example, `rayservice-sample-head-svc` and `rayservice-sample-serve-svc`.\n",
     "\n",
+    "> 🧠 **What do these services do?**\n",
+    "\n",
+    "- **`rayservice-sample-head-svc`**  \n",
+    "  This service exposes the Ray **head node**, allowing access to:\n",
```

Comment:
"This service points to the head pod of the active RayCluster and is typically used to view the Ray dashboard."

In addition, Ray community doesn't encourage users to use Ray client, so I would suggest remove the line of "- **Ray Client** on port `10001`".